app/scripts/directives/configEditor.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('configEditor', function(APIService, DataService) {
+    return {
+      restrict: 'E',
+      scope: {
+        mobileClient: '<',
+      },
+      templateUrl: 'views/config-editor.html',
+      controller: function($scope) {
```

Comment:
Unless you specifically need `$scope` I would use `$ctrl`

Reply:
when I change this to `$ctrl` the prettyConfig is no longer available on the template.